proprietary/tokens/src/components/ams/description-list.tokens.json

```
@@ -0,0 +1,20 @@
+{
+  "ams": {
+    "description-list": {
+      "color": { "value": "{ams.color.primary-black}" },
+      "font-family": { "value": "{ams.text.font-family}" },
+      "font-size": { "value": "{ams.text.level.5.font-size}" },
+      "font-weight": { "value": "{ams.text.font-weight.normal}" },
+      "gap": { "value": "{ams.space.stack.md}" },
+      "inverse-color": { "value": "{ams.color.primary-white}" },
+      "line-height": { "value": "{ams.text.level.5.line-height}" },
+      "row": {
+        "gap": { "value": "{ams.space.inside.md}" }
```

If we keep the row, this should use the Stack scale as well.
```suggestion
        "gap": { "value": "{ams.space.stack.md}" }
```